.gitignore

```diff
@@ -9,9 +9,9 @@
 *.bat
 *.compiled
 *.ini
+*.dcu
+__history/
+__recovery/
+Win32/
 bin/
 lib/
```

Comment:
@nunopicado What do you think to add here some binaries files as `*.res`, `*.identcache`, `*.bpl` and*.proj`?